operator/v1/types.go

```diff
@@ -98,22 +98,22 @@ type KlusterletDeployOption struct {
 
 // ClusterManagerDeployOption describes the deploy options for cluster-manager
 type ClusterManagerDeployOption struct {
-	// Mode can be Default or Detached.
+	// Mode can be Default or Hosted.
 	// In Default mode, the Hub is installed as a whole and all parts of Hub are deployed in the same cluster.
-	// In Detached mode, only crd and configurations are installed on one cluster(defined as hub-cluster). Controllers run in another
+	// In Hosted mode, only crd and configurations are installed on one cluster(defined as hub-cluster). Controllers run in another
 	// cluster (defined as management-cluster) and connect to the hub with the kubeconfig in secret of "external-hub-kubeconfig"(a kubeconfig
 	// of hub-cluster with cluster-admin permission).
 	// Note: Do not modify the Mode field once it's applied.
 	// +required
 	// +default=Default
 	// +kubebuilder:validation:Required
 	// +kubebuilder:default=Default
-	// +kubebuilder:validation:Enum=Default;Detached
+	// +kubebuilder:validation:Enum=Default;Detached;Hosted
```

Comment:
Do we still need the `Detached` value for cluster manager?

Reply:
That's a good point! Removed Detached mode.